compose/service.py

```diff
@@ -399,6 +401,12 @@ def _get_container_create_options(self, override_options, one_off=False):
             self.containers(stopped=True, one_off=one_off),
             one_off)
 
+        if 'labels' in container_options:
+            labels_dict = ['io.docker.compose:' + label for label in container_options['labels']]
```

Comment:
We should not  be namespacing labels here, it should be left up to the developer.  

We will likely use a namespace for our internal labels in #1066, but any user specified labels should be unmodified I think.